src/App.tsx

```diff
@@ -1,223 +1,90 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React, { useContext, useEffect, useRef } from 'react';
+import React, {
+  useContext,
+  useEffect,
+  useRef,
+  useState,
+} from 'react';
 import { AuthContext } from './components/Auth/AuthContext';
+import { Footer } from './components/Footer';
+import { Header } from './components/Header';
+import { TodoList } from './components/TodoList';
+import { ErrorNotification } from './components/ErrorNotification';
+import { Todo } from './types/Todo';
+import { FilterType } from './types/FilterType';
+import { getTodos } from './api/todos';
 
 export const App: React.FC = () => {
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [errorStatus, setErrorStatus] = useState<boolean>(false);
+  const [filterBy, setFilterBy] = useState<FilterType>(FilterType.ALL);
+  const [errorText] = useState<string>('');
+
   const user = useContext(AuthContext);
   const newTodoField = useRef<HTMLInputElement>(null);
 
+  if (errorStatus) {
+    setTimeout(() => {
+      setErrorStatus(false);
+    }, 3000);
+  }
```

Comment:
you need to create setTimeout on the first render ErrorNotification and clear timeout on unmount